app/src/panels/relational-variable/multiple-relation.vue

```diff
@@ -91,6 +91,19 @@ function deleteItem(elem: Record<string, any>) {
 	align-items: center;
 	gap: var(--v-sheet-padding);
 
+	> .v-button {
+		:deep(.button) {
+			--v-button-width: auto;
```

Comment:
I think we could just use a `<button>` instead of a `v-button` with a bunch of style overrides 🤔

Reply:
Makes sense! Updated to `<button>` and removed other unused styles.